node/node.go

```diff
@@ -152,6 +152,14 @@ func CustomReactors(reactors map[string]p2p.Reactor) Option {
 				n.sw.RemoveReactor(name, existingReactor)
 			}
 			n.sw.AddReactor(name, reactor)
+			// register the new channels to the nodeInfo
+			for _, chDesc := range reactor.GetChannels() {
+				// these functions only error when the channel is already
+				// registered. This may be the case when we are replacing
+				// existing reactors so we can ignore these
```

Comment:
do we have to do any cleanup if we *are* replacing a channe?

Reply:
I don't think the reactor would have started yet